sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ext/opentelemetry_span/_schema.py

```diff
@@ -28,7 +26,7 @@ class OpenTelemetrySchema:
             "x-ms-client-request-id": "az.client_request_id",
             "x-ms-request-id": "az.service_request_id",
             "http.user_agent": "user_agent.original",
-            "messaging_bus.destination": "messaging.destination.name",
+            "message_bus.destination": "messaging.destination.name",
```

Comment:
will this cause issues for folks using `messaging_bus` previously ?

Reply:
Nah, it shouldn't. The addition of the `OpenTelemetrySchema` class is still unreleased. And `messaging_bus.destination` was mainly just a typo as the intent for this dict is to remap keys used in the SDKs to converge on OTel defined keys. And nothing uses `messaging_bus.destination` (`message_bus.destination` is used however).